ui/app/mirrors/create/schema.ts

```diff
@@ -38,34 +38,22 @@ export const tableMappingSchema = z
   });
 
 export const cdcSchema = z.object({
-  source: z.object(
-    {
-      name: z.string().min(1),
-      type: z.any(),
-      config: z.any(),
-    },
-    { required_error: 'Source peer is required' }
-  ),
-  destination: z.object(
-    {
-      name: z.string().min(1),
-      type: z.any(),
-      config: z.any(),
-    },
-    { required_error: 'Destination peer is required' }
-  ),
+  sourceName: z.string({ required_error: 'Source peer is required' }).min(1),
+  destinationName: z
+    .string({ required_error: 'Destination peer is required' })
+    .min(1),
   doInitialCopy: z.boolean().optional(),
   publicationName: z
     .string({
       invalid_type_error: 'Publication name must be a string',
     })
-    .max(255, 'Publication name must be less than 255 characters')
+    .max(255, 'Publication name must be less than 256 characters')
```

Comment:
these seem arbitrary, PG only supports 63 byte identifiers without recompiling

Reply:
yes, but these changes are to have text match code